unsloth/save.py

```diff
@@ -2347,3 +2287,27 @@ def patch_saving_functions(model, vision = False):
     pass
     return model
 pass
+
```

Comment:
Ok this is a reasonable function - we can probably just include this if that works :)